edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Integration.Test/test-configs/module-priority-test-config.json

```diff
@@ -395,5 +399,162 @@
         }
       ]
     }
+  },
+  {
+    "deploymentConfig": {
+      "schemaVersion": "1.0",
+      "runtime": {
+        "type": "docker",
+        "settings": {
+          "minDockerVersion": "v1.25",
+          "loggingOptions": "",
+          "registryCredentials": {}
+        }
+      },
+      "systemModules": {
+        "edgeAgent": {
+          "type": "docker",
+          "settings": {
+            "image": "mcr.microsoft.com/azureiotedge-agent:1.0",
+            "createOptions": "{}"
+          },
+          "startupOrder": 2,
+          "env": {
+            "ModuleUpdateMode": {
+              "value": "WaitForAllPulls"
+            }
+          }
+        },
+        "edgeHub": {
+          "type": "docker",
+          "settings": {
+            "image": "mcr.microsoft.com/azureiotedge-hub:1.0",
+            "createOptions": "{\"HostConfig\":{\"PortBindings\":{\"8883/tcp\":[{\"HostPort\":\"8883\"}],\"443/tcp\":[{\"HostPort\":\"443\"}],\"5671/tcp\":[{\"HostPort\":\"5671\"}]},\"StopTimeout\":60}}"
+          },
+          "env": {},
+          "startupOrder": 0,
+          "status": "running",
+          "restartPolicy": "always"
+        }
+      },
+      "modules": {
+        "tempSensor": {
+          "type": "docker",
+          "settings": {
+            "image": "edgebuilds.azurecr.io/microsoft/azureiotedge-simulated-temperature-sensor:20191106.7-linux-amd64",
+            "createOptions": ""
+          },
+          "startupOrder": 1,
+          "status": "running",
+          "restartPolicy": "always"
+        },
+        "tempSensor2": {
+          "type": "docker",
+          "settings": {
+            "image": "edgebuilds.azurecr.io/microsoft/azureiotedge-simulated-temperature-sensor:20191106.7-linux-amd64",
+            "createOptions": ""
+          },
+          "status": "running",
+          "restartPolicy": "always"
+        }
+      }
+    },
+    "runtimeInfo": {
+      "modules": {
+        "edgeAgent": {
+          "exitCode": 0,
+          "runtimeStatus": "running",
+          "status": "running",
+          "restartPolicy": "always",
+          "startupOrder": 2,
+          "type": "docker",
+          "settings": {
+            "image": "mcr.microsoft.com/azureiotedge-agent:2.0",
```

Comment:
2.0 doesn't exist...is this intentional?

Reply:
2.0 is used above in an existing test that I didn't write. Image is never actually pulled, this integration test is testing the ordering that the commands are executed.